emails/send_newsletter-0524.tsx

```diff
@@ -0,0 +1,33 @@
+import SendGrid from "@sendgrid/mail";
+import { render } from "@react-email/components";
+import _Newsletter0524 from "./_newsletter-0524";
+import { REPLY_EMAIL } from "@/lib/email/utils";
+SendGrid.setApiKey(process.env.SENDGRID_API_KEY);
+
+export interface SendNewsletter0524Props {
+  email: string;
+  unsubscribeUrl: string;
+}
+
+export async function sendNewsletter0524({
+  email,
+  unsubscribeUrl,
+}: SendNewsletter0524Props) {
+  try {
+    if (!email) throw new Error("No email provided");
+    if (!unsubscribeUrl) throw new Error("No unsubscribeUrl provided");
+
+    await SendGrid.send({
+      from: {
+        email: REPLY_EMAIL,
+        name: "Hawaiians in Tech",
+      },
+      to: email,
+      subject: `🌺 Huuui! Hawaiian in Tech May Newsletter`,
+      html: render(<_Newsletter0524 unsubscribeUrl={unsubscribeUrl} />),
```

Comment:
If weʻre going to merge this, can we change that `_Newsletter` → `Newsletter`? Or at least comment a TODO because I donʻt want us to make a habit of non-conventional component naming. My fault for shortsightedly naming this one 🤦

Reply:
updated before merge